### DISCUSSION TITLE
GeneActivity function and TSS location

### DISCUSSION THREAD

**Discussion by skilpinen** [stuart-lab/signac#508] [Q&A] [answered]
Hi,

I might have missed this but could you elaborate a bit how GeneActivity() as well as LinkPeaks() define TSS position exactly? Given that for some genes if one fetches TSS location for example from BioMart there are multiple TSS locations as those might vary given some transcript variants. I have need to find same region used by these functions, but I haven't found explicit information on what data source and logic was used for TSS coordinates, perhaps this info could be added to the documentation if it is not there already.

**Comment by timoast** [comment, marked as answer]
By default the TSS positions are defined by the set of gene annotations stored in the object. We collapse each gene down to the longest transcript and take the start of that transcript as the TSS for the gene. If you want to use a different set of regions, you can use the `tss.positions` parameter in `TSSEnrichment`. If you want to use a different set of regions for `GeneActivity()` you can just run `FeatureMatrix()` on your own set of regions instead.
